test/dummy_luascript.lua

```diff
@@ -0,0 +1,11 @@
+var = 18
+
+if var>18 then
+   print("Racheal gets 2 chocolates")
+end
+elseif var==18 then
```

Comment:
This isn't valid lua (intentional?)

Reply:
Its just dummy, nothing to do with what is written inside.